workspaces/datakube-prod/backend.cfg

```diff
@@ -0,0 +1,4 @@
+bucket="dea-devs-tfstate"
```

Comment:
Will only leave one comment on the names and we are in progress fixing that up in #7

Reply:
It's just copied from the prior config, should be fixed when making names generic.